Src/CSharpier.Cli/IgnoreFile.cs

```diff
@@ -39,58 +37,50 @@ public bool IsIgnored(string filePath)
         CancellationToken cancellationToken
     )
     {
-        DebugLogger.Log("Creating IgnoreFile for " + baseDirectoryPath);
         var ignoreFilePaths = FindIgnorePaths(baseDirectoryPath, fileSystem);
         if (ignoreFilePaths.Count == 0)
         {
             var ignore = new IgnoreWithBasePath(baseDirectoryPath);
-            foreach (var name in alwaysIgnored)
-            {
-                ignore.Add(name);
-            }
+            AddDefaultRules(ignore);
             return new IgnoreFile([ignore]);
         }
 
         var ignores = new List<IgnoreWithBasePath>();
         foreach (var ignoreFilePath in ignoreFilePaths)
         {
             var ignore = new IgnoreWithBasePath(Path.GetDirectoryName(ignoreFilePath)!);
+            AddDefaultRules(ignore);
+
+            var content = await fileSystem.File.ReadAllTextAsync(
+                ignoreFilePath,
+                cancellationToken
+            );
+
+            var (positives, negatives) = GitignoreParserNet.GitignoreParser.Parse(content, true);
+
+            ignore.AddPositives(positives.Merged);
+            ignore.AddNegatives(negatives.Merged);
+            
             ignores.Add(ignore);
-            foreach (var name in alwaysIgnored)
-            {
-                ignore.Add(name);
-            }
-            foreach (
-                var line in await fileSystem.File.ReadAllLinesAsync(
-                    ignoreFilePath,
-                    cancellationToken
-                )
-            )
-            {
-                if (string.IsNullOrWhiteSpace(line))
-                {
-                    continue;
-                }
-                try
-                {
-                    ignore.Add(line);
-                }
-                catch (Exception ex)
-                {
-                    throw new InvalidIgnoreFileException(
-                        $"""
-                        The .csharpierignore file at {ignoreFilePath} could not be parsed due to the following line:
-                        {line}
-                        """,
-                        ex
-                    );
-                }
-            }
         }
 
         return new IgnoreFile(ignores);
     }
 
+    private static void AddDefaultRules(IgnoreWithBasePath ignore)
+    {
+        var stringBuilder = new StringBuilder();
+        foreach (var s in alwaysIgnored)
+        {
+            stringBuilder.AppendLine(s);
+        }
```

Comment:
`alwaysIgnored` could be a raw string to simplify this and for a minor performance improvement.